src/pages/index.tsx

```diff
@@ -261,151 +126,76 @@ const Home: NextPage<HomePageProps> = ({ marketplace }) => {
+                </li>
+                <li className='flex justify-between w-full px-4 py-2 mb-1 bg-gray-800 rounded-md cursor-pointer hover:bg-gray-800'>
                   <h4>Unlisted</h4>
-                  <span className='text-sm text-gray-500'>0</span>
-                </div>
-              </div>
-              <div className='flex flex-col flex-grow gap-4'>
-                {sidebar.data?.creator.attributeGroups.map(
-                  ({ name: group, variants }, index) => (
-                    <div className='flex flex-col flex-grow gap-2' key={group}>
-                      <label className='label'>
-                        {group.charAt(0).toUpperCase() + group.slice(1)}
-                      </label>
-
-                      <Controller
-                        control={control}
-                        name={`attributes.${index}`}
-                        defaultValue={{ traitType: group, values: [] }}
-                        render={({ field: { onChange, value } }) => {
-                          return (
-                            <Select
-                              value={value.values}
-                              isMulti
-                              className='select-base-theme'
-                              classNamePrefix='base'
-                              onChange={(next: ValueType<OptionType>) => {
-                                onChange({ traitType: group, values: next })
-                              }}
-                              formatOptionLabel={(
-                                elem: ValueType<OptionType>
-                              ) =>
-                                elem.label
-                                  .split(' ')
-                                  .slice(0, -1)
-                                  .join(' ') +
-                                ' (' +
-                                elem.label.split(' ').at(-1) +
-                                ')'
-                              }
-                              options={
-                                variants.map(({ name, count }) => ({
-                                  value: name,
-                                  label: `${name} ${count}`,
-                                })) as OptionsType<OptionType>
-                              }
-                            />
-                          )
-                        }}
-                      />
-                    </div>
-                  )
-                )}
-              </div>
+                </li>
+              </ul>
+              <label className="label mb-2">Creators</label>
+              <ul className="flex flex-col flex-grow mb-6">
+                <li>
+                  <Link href={`/creators/${marketplace.ownerAddress}`}>
+                    <a className='flex justify-between w-full px-4 py-2 mb-1 rounded-md cursor-pointer hover:bg-gray-800'>
+                      <h4>{truncateAddress(marketplace.ownerAddress)}</h4>
```

Comment:
potentially cool to convert wallets to a class and get truncated addresses with `marketplace.owner.truncatedAddress()`